app.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "ride-my-way",
+  "scripts": {},
+  "env": {},
+  "formation": {},
+  "addons": [],
+  "buildpacks": []
+}
```

Comment:
leave an extra line

Reply:
Lol okay

Reply:
There's an extra line Github removes it when displaying. If there wasn't you would see a red arrow under need the file.

Reply:
Okay, Got it

Reply:
Please you need to rebase from develop